.github/workflows/lint.yml

```diff
@@ -28,6 +28,22 @@ jobs:
         run: pip install --user yamllint
       - name: Run yamllint
         run: yamllint .
+  black:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Not sure, do we really need to run this with multiple versions of Python?

Reply:
I don't think so, in my opinion having only 3.11 is just fine!

Reply:
@kbabioch Any thoughts on my last question about pylint?

Reply:
I'm fine with only having 3.11 for linting as well. Sorry for the late reply, somehow I ran into "out of time" errors ;-).